Sources/DashX/DashXClient.swift

```diff
@@ -588,4 +607,23 @@ public class DashXClient {
     public func requestLocationPermission() {
         CLLocationManager().requestWhenInUseAuthorization()
     }
+    
+    // MARK: - Get ID of Notification
+    
+    private func getNotificationID(_ message: [AnyHashable : Any]) -> String? {
```

Comment:
We are doing something similar in `application(_:didReceiveRemoteNotification:fetchCompletionHandler:)`  to parse dashx notification payload. 


```
if let jsonData = stringData.data(using: .utf8) {
    do {
        let dashxData = try JSONDecoder().decode(DashXNotificationData.self, from: jsonData)
    } catch {
        DashXLog.d(tag: #function, "Unable to parse DashX notification data \(error)")
    }
}
```
Is it possible to extend `[AnyHashable : Any]` type so we could just do 
- `message.dashxData` 
- `message.dashxNotificationId`  which would internally just call `message.dashxData.id`